allensdk/brain_observatory/behavior/write_nwb/_schemas.py

```diff
@@ -134,6 +134,8 @@ class Meta:
     output_path = String(required=True, validate=check_write_access_overwrite,
                          description='write outputs to here')
 
+    skip_eye_tracking = Boolean(required=True, default=False,
```

Comment:
This is definitely going to cause problems.

The current LIMs strategy for writing behavior + ophys NWB files doesn't write a `skip_eye_tracking` field in the `input.json`.

I think to get it to play nice you will at least need to set `required=False`. And just to be extra safe:

- [ ] Try running the updated `write_nwb` __main__ with an input JSON file that doesn't have a `skip_eye_tracking` field.

Reply:
I did check, and it runs fine without providing `skip_eye_tracking`. That's the behavior I would expect since a default value is given.